scripts/ripe-atlas

```diff
@@ -1,94 +1,10 @@
 #!/usr/bin/env python
-
-import importlib
-import os
-import pkgutil
-import re
 import sys
 
-from ripe.atlas.tools import commands
+from ripe.atlas.tools.script import RipeAtlas
```

Comment:
Yeah I'm not thrilled about it, so I'd rather not create an essentially empty pointer file unless there's an outside reason to do so.  So for my vote: no-split.